### PR TITLE
OCA add: Nextcloud

### PR DESCRIPTION
**Pull request by hmorris3293**
##Description
A Nextcloud One-Click App

## Type of Change
 * Non breaking change ('update', 'change')
 
## Note to Reviewers
TODO:
 
 * [x]  Convert PNGs to SVGs
 * [x]  Set ordinal location of this OCA

### CODE REVIEW

File: packages/manager/public/assets/nextcloud.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2.4 8.8-8 8.8h-5.6c-5.6 0-8.8-2.4-8.8-8v-13.7c0-6.4 1.6-11.2 1.6-11.2h-.8s-15.3 36.9-61 36.9c-47.4 0-76.3-37.8-77.1-92.4-.9-54.5 31.2-91.4 77.8-91.4zm.8 163.8c29.7 0 57-20.9 57-71.5 0-36.1-18.5-70.7-56.2-70.7-31.3 0-57.4 25.7-57 70.7.3 43.4 23.2 71.5 56.2 71.5z"/><path d="m273.3 1612.8h6.4c5.6 0 8.8-3.2 8.8-8.8v-172.4c0-27.3 29.7-46.8 63.4-46.8s63.4 19.5 63.4 46.8v172.4c0 5.6 3.2 8.8 8.8 8.8h6.4c5.6 0 8-3.2 8-8.8v-173.5c0-45.8-45.8-68.3-87.5-68.3-40.2 0-85.9 22.5-85.9 68.3v173.5c.1 5.6 2.5 8.8 8.2 8.8z"/><path d="m1613.6 1436.1h-6.4c-5.6 0-8.8 3.2-8.8 8.8v97.2c0 27.3-17.7 52.2-52.2 52.2-33.7 0-52.2-24.9-52.2-52.2v-97.2c0-5.6-3.2-8.8-8.8-8.8h-6.4c-5.6 0-8 3.2-8 8.8v103.6c0 45.8 33.7 68.3 75.5 68.3s75.5-22.5 75.5-68.3v-103.6c.6-5.6-2.6-8.8-8.2-8.8z"/><path d="m818.6 1434.8c-2 .3-3.9 1.6-5.7 3.8l-56.8 67.7-56.8-67.7c-1.8-2.2-3.9-3.3-6-3.5s-4.3.6-6.5 2.4l-4.9 4.1c-4.3 3.6-4.1 7.6-.5 11.9l59.5 70.9-39.5 47s0 .1-.1.1l-19.9 23.7c-3.6 4.3-3.2 8.8 1.1 12.5l4.9 4.1c4.3 3.6 8.2 2.7 11.8-1.6l56.8-67.7 36.8 43.9s.1 0 .1.1l19.9 23.7c3.6 4.3 8.1 4.7 12.4 1.1l4.9-4.1c4.3-3.6 4.1-7.6.5-11.9l-59.5-70.9 39.5-47s0-.1.1-.1l19.9-23.7c3.6-4.3 3.2-8.8-1.1-12.5l-4.9-4.1c-2-1.9-4-2.5-6-2.2z"/></g></g></svg>
```
Comment (abalarin) [inline packages/manager/public/assets/nextcloud.svg:1]: best practice is to have a new line at the end of file

File: packages/manager/public/assets/nextcloud_color.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...2.4 8.8-8 8.8h-5.6c-5.6 0-8.8-2.4-8.8-8v-13.7c0-6.4 1.6-11.2 1.6-11.2h-.8s-15.3 36.9-61 36.9c-47.4 0-76.3-37.8-77.1-92.4-.9-54.5 31.2-91.4 77.8-91.4zm.8 163.8c29.7 0 57-20.9 57-71.5 0-36.1-18.5-70.7-56.2-70.7-31.3 0-57.4 25.7-57 70.7.3 43.4 23.2 71.5 56.2 71.5z"/><path d="m273.3 1612.8h6.4c5.6 0 8.8-3.2 8.8-8.8v-172.4c0-27.3 29.7-46.8 63.4-46.8s63.4 19.5 63.4 46.8v172.4c0 5.6 3.2 8.8 8.8 8.8h6.4c5.6 0 8-3.2 8-8.8v-173.5c0-45.8-45.8-68.3-87.5-68.3-40.2 0-85.9 22.5-85.9 68.3v173.5c.1 5.6 2.5 8.8 8.2 8.8z"/><path d="m1613.6 1436.1h-6.4c-5.6 0-8.8 3.2-8.8 8.8v97.2c0 27.3-17.7 52.2-52.2 52.2-33.7 0-52.2-24.9-52.2-52.2v-97.2c0-5.6-3.2-8.8-8.8-8.8h-6.4c-5.6 0-8 3.2-8 8.8v103.6c0 45.8 33.7 68.3 75.5 68.3s75.5-22.5 75.5-68.3v-103.6c.6-5.6-2.6-8.8-8.2-8.8z"/><path d="m818.6 1434.8c-2 .3-3.9 1.6-5.7 3.8l-56.8 67.7-56.8-67.7c-1.8-2.2-3.9-3.3-6-3.5s-4.3.6-6.5 2.4l-4.9 4.1c-4.3 3.6-4.1 7.6-.5 11.9l59.5 70.9-39.5 47s0 .1-.1.1l-19.9 23.7c-3.6 4.3-3.2 8.8 1.1 12.5l4.9 4.1c4.3 3.6 8.2 2.7 11.8-1.6l56.8-67.7 36.8 43.9s.1 0 .1.1l19.9 23.7c3.6 4.3 8.1 4.7 12.4 1.1l4.9-4.1c4.3-3.6 4.1-7.6.5-11.9l-59.5-70.9 39.5-47s0-.1.1-.1l19.9-23.7c3.6-4.3 3.2-8.8-1.1-12.5l-4.9-4.1c-2-1.9-4-2.5-6-2.2z"/></g></g></svg>
```
Comment (abalarin) [inline packages/manager/public/assets/nextcloud_color.svg:1]: same here with new line at the end of the file

File: packages/manager/public/assets/nextcloud.svg (hunk 1 above)
Comment (hmorris3293) [reply packages/manager/public/assets/nextcloud.svg]: updated

File: packages/manager/public/assets/nextcloud_color.svg (hunk 2 above)
Comment (hmorris3293) [reply packages/manager/public/assets/nextcloud_color.svg]: updated

**Comment by leslitagordita** [comment]
@hmorris3293 Here is the PR for the OCA guide: https://github.com/linode/docs/pull/3398. When I was testing I noticed that if you don't provide a Domain name, you cannot access the OCA using the Linode's IP address. I think for this reason, domain name should be marked as a required field. If I'm missing something here, please let me know. In the doc, I labeled the field as required and also listed creating an APIv4 token as a prerequisite.

**Pull request merged**